content/tutorial/02-sveltekit/02-routing/02-layouts/README.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Layouts
+---
+
+Different routes of your app will often share common UI. Instead of repeating it in each `+page.svelte` component, we can use a `+layout.svelte` component that applies to all routes in the same directory.
```

Comment:
```suggestion
Different routes of your app will often share common UI. Instead of repeating the common portions in each `+page.svelte` component, we can use a `+layout.svelte` component that applies to all routes in the same directory.
```